tests/test_modeling_gpt_neo.py

```diff
@@ -238,6 +239,17 @@ def create_and_check_lm_head_model(self, config, input_ids, input_mask, head_mas
         self.parent.assertEqual(result.loss.shape, ())
         self.parent.assertEqual(result.logits.shape, (self.batch_size, self.seq_length, self.vocab_size))
 
+    def create_and_check_gpt_neo_for_sequence_classification(
+        self, config, input_ids, input_mask, head_mask, token_type_ids, mc_token_ids, sequence_labels, *args
+    ):
+        config.num_labels = self.num_labels
+        model = GPTNeoForSequenceClassification(config)
+        model.to(torch_device)
+        model.eval()
+        print(config.num_labels, sequence_labels.size())
```

Comment:
I think this is a leftover from debugging :-)

Reply:
Hi @sgugger,
you are right, I actually copied the code from this GPT2 test file which is having the same debugging mistake
https://github.com/huggingface/transformers/blob/afe479adb5474250215438fe27db9dc9dbbbde09/tests/test_modeling_gpt2.py#L364
Shall we also fix it or keep it as it is?

Reply:
Could you remove it too? That would be great!

Reply:
Sure, I will remove it!